### PR TITLE
Feature/month value object

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

File: src/Nox.Types.EntityFramework/MonthConverter.cs
Comment (rochar) [inline src/Nox.Types.EntityFramework/MonthConverter.cs:1]: Can we move this class to a subFolder called ./Types/Month? keeping the namespace, and having the same folder struct in Nox.Types,NoxTypes.EntityFramework, and Nox.Types.Tests?

We are working on setting this conventions, maybe you can start following what will be propose, thks a lot

File: tests/Nox.Types.Tests/NoxMonthTests.cs
Comment (rochar) [inline tests/Nox.Types.Tests/NoxMonthTests.cs:1]: Can we move this class to a subFolder called ./Types/Month? keeping the namespace, and having the same folder struct in Nox.Types,NoxTypes.EntityFramework, and Nox.Types.Tests?

And rename the class to MonthTests

We are working on setting this conventions, maybe you can start following what will be propose, thks a lot

**Review by rochar — CHANGES_REQUESTED** [review]
looks good, just a few comments
thks

File: src/Nox.Types.EntityFramework/MonthConverter.cs
Comment (msadikkoseaupa) [reply src/Nox.Types.EntityFramework/MonthConverter.cs:1]: folder structure was updated as desired

File: tests/Nox.Types.Tests/NoxMonthTests.cs
Comment (msadikkoseaupa) [reply tests/Nox.Types.Tests/NoxMonthTests.cs:1]: folder structure was updated as desired

**Review by rochar — APPROVED** [review]

**Pull request merged**